### PR TITLE
Integrate User First/Last Name UI

### PR DESCRIPTION
**Pull request by ousmaneo**
Integrate Graylog User First and Last Name UI into cloud-4.0

See #10158

### CODE REVIEW

**Review by danotorrey — APPROVED** [review]
I can't say much about the code, but I test it, and it works well 👍

File: graylog2-web-interface/src/components/users/UserCreate/FullNameFormGroup.jsx [hunk 1]
```diff
@@ -19,12 +19,12 @@ import * as React from 'react';
 
 import { FormikFormGroup } from 'components/common';
 
-const FullNameFormGroup = () => (
-  <FormikFormGroup label="Full Name"
-                   name="full_name"
+const LastNameFormGroup = () => (
```
Comment (edmundoa) [inline graylog2-web-interface/src/components/users/UserCreate/FullNameFormGroup.jsx:22]: Is this component still needed? The same folder has a new `LastNameFormGroup` component doing exactly the same.

**Review by edmundoa — CHANGES_REQUESTED** [review]
A small nitpick, otherwise it looks good to me.

File: graylog2-web-interface/src/components/users/UserCreate/FullNameFormGroup.jsx (hunk 1 above)
Comment (ousmaneo) [reply graylog2-web-interface/src/components/users/UserCreate/FullNameFormGroup.jsx:22]: Thank you for pointing this out, this not needed anymore.

**Review by edmundoa — APPROVED** [review]
LGTM 👍

**Pull request merged**